hw/ip/aes/data/aes.hjson

```diff
@@ -15,6 +15,12 @@
       desc:    "Number registers for key",
       local:   "true"
     },
+    { name:    "NumRegsIv",
+      type:    "int",
+      default: "4",
```

Comment:
I didn't think IV width was configurable per CTR standard? Do we ever expect
this to be anything other than 4?

Reply:
Yes, you are right it is not actually a configurable parameter and there will never be more than 4 IV registers. But if I don't use a parameter and put a hardcoded value in `aes.hjson` the regtool generates a warning every time it is run:
```
WARNING: IV multireg count is integer. It is recommended to use Parameter.
```
I would thus leave it as a parameter. But I added a comment in the hjson that these are local and not actually configurable.

Reply:
@eunchan is his method the best solution here?

Reply:
Looks like the best solution. It would be good if reggen adds a field `type` in the param list to indicate whether it is localparam or configurable parameter.